requirements/runtime.txt

```diff
@@ -1,3 +1,4 @@
+faster-coco-eval
```

Comment:
I think it's best to make this dependency optional, to prevent affecting the previous usage pattern.

Reply:
done